utils/simple_dict.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
```

Comment:
Using `utils` is for espnet1 style, where we are not using mostly (and recommend not to use, since we actually remove the checks to utils in espnet2 case). For this function, you might add it to `egs2/TEMPLATE/asr1/scripts` instead of here

Reply:
Aha, didn't know that! Thanks for pointing that out!! My intention of simple_dict.sh was to design a shell library that can be used to other projects too (Python's basic data structure is list and dict, so I thought we needed dict on shell for many python programmers, whereas list is simple enough already, so didn't implement it). So, (1) will it be a good idea to maintain a shell library in the first place, and (2) if so, where should I put it for the uses in espnet2?

Reply:
Sorry for the late response! It would be good to have utility functions, but in that case, I would still recommend putting the one in `egs2/TEMPLATE/asr1/scripts/utils` as we won't include utils as in the `path.sh` and almost all the tasks have symlink to asr1

Reply:
Or if @sw005320 would have other ideas, please let us know

Reply:
For (2), as Jiatong suggested, we usually put the recipe-related utility espne2 scripts in `egs2/TEMPLATE/asr1/scripts `

Reply:
Yeah, I'll relocate that file in the https://github.com/espnet/espnet/pull/5441 PR! I don't want the merge conflict to be too severe (because that PR and this PR has diverged a bit), so I'll try to merge this PR ASAP, and then modify that in the follow-up PR.